data-pipeline/content-cache-updater/pom.xml

```diff
@@ -12,7 +12,7 @@
 
     <groupId>org.ekstep.ecosystem.jobs</groupId>
     <artifactId>content-cache-updater</artifactId>
-    <version>0.1.6</version>
+    <version>0.0.2</version>
```

Comment:
Is the version right? shouldn't it be 0.1.7?

Reply:
The content-cache-update job was split from RedisUpdater job. However, the version was not reset to 0.0.1 to begin with. I have corrected this version.